### DISCUSSION TITLE
Populate single select with values from another collection

### DISCUSSION THREAD

**Discussion by ggros** [AntlerVC/firetable#701] [Support / Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
If you have a table with let's say orders tied to a client, I would need to be able to see the client name instead of the id.
Also ideally we should be able to edit the client an order is tied to.
I do not know if using Algolia connect field type would solve this issue but I would not want to setup Algolia just for this.

**Describe the solution you'd like**
I would like to be able to use the single Select option, and describe the link needed to populate the Select.

**Describe alternatives you've considered**

**Additional context**

**Comment by notsidney** [comment]
Hi @ggros, are the order documents:
1. Created with the client document ID in a field already, outside of Firetable?
2. Or did you want to be able to link orders to a client directly on Firetable?

**If 1 and you don’t need to change which client is linked to an order on Firetable:**
- You can create a Derivative field, which runs a cloud function that puts a value in that field based on the client ID. So it reads the client document, then returns that client’s name.
- You can do the same thing but either with your own cloud function or whatever you currently use to create orders

**If 2:**
This is what the Algolia/Connect Table column is used for. Please note that we do have a specific data structure for this field, so you should create a new column/field for this.

***

Unfortunately, we currently do not have documentation on how to set up either of those column types, but @shamsmosowi can assist here.

**Comment by harinij** [comment, marked as answer]
This is now supported via connector field. https://docs.rowy.io/field-types/connector
